packages/sourcecred/src/core/ledger/ledger.js

```diff
@@ -847,6 +921,80 @@ export class Ledger {
     return this._grainIntegrationStatuses.get(id);
   }
 
+  /**
+   *   This function can be called while accounting is disabled to update
+   *   the ledger's external currency configuration. Note that this will also
+   *   deactivate identities missing a payout address that matches the new
+   *   configuration when accounting is disabled.
```

Comment:
Would be good to add the behavior of this when accounting is enable. Throws? No effect?